tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
Yeah, it's an issue. I was just wondering if waiting for .app-reviews rather than .loaded resolves that issue.